portal/templates/redesign/teach_new/onboarding_school.html

```diff
@@ -143,5 +213,7 @@ <h4 class="col-sm-offset-2">Everything you need to start coding with your class
         }
     }
 </script>
+{% endif %}
+{% endif %}
```

Comment:
Either something bad has happened with your indentation, or you have a reduplication here.